docker/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.3'
+
+services:
+  db:
+    image: mariadb
+    restart: always
+    environment:
+      - MARIADB_RANDOM_ROOT_PASSWORD=yes
+      - MARIADB_USER=tavern
+      - MARIADB_PASSWORD=plodder09*arouse
```

Comment:
I'd try to avoid hardcoded passwords.
Instead you can use an `.env` file to generate a random password:
.env
```bash
export DB_PASSWORD=`pwgen -Bs1 12`
```

```yaml
     - MARIADB_PASSWORD: ${DB_PASSWORD}
```

https://stackoverflow.com/a/39713771

Reply:
NVM docker secrets is a better solution.
https://github.com/KCarretto/realm/pull/314/files#r1351358013

You could still use an env file to generate a random password

Reply:
> I'd try to avoid hardcoded passwords. Instead you can use an `.env` file to generate a random password

This is also a viable solution, either this or docker compose secrets are sufficient